tools/wptserve/wptserve/server.py

```diff
@@ -190,8 +187,8 @@ def __init__(self, server_address, request_handler_cls,
         else:
             hostname_port = ("",server_address[1])
 
-        #super doesn't work here because BaseHTTPServer.HTTPServer is old-style
-        BaseHTTPServer.HTTPServer.__init__(self, hostname_port, request_handler_cls, **kwargs)
+        #super doesn't work here because http.server.HTTPServer is old-style
```

Comment:
Old-style classes don't exist any more, so this comment doesn't make sense (I think it's fine to leave the code as-is fwiw).

Reply:
Removed comment.